MapboxMobileEvents/MMEDate.m

```diff
@@ -55,6 +55,7 @@ + (NSDateFormatter *)HTTPDateFormatter {
     if (!_httpDateFormatter) {
         _httpDateFormatter = [[NSDateFormatter alloc] init];
         [_httpDateFormatter setDateFormat:@"E, d MMM yyyy HH:mm:ss Z"];
+        [_httpDateFormatter setTimeZone:[NSTimeZone timeZoneWithName:@"PST"]];
```

Comment:
@alfwatt @rclee ,

One of the errors I got once the CI was up and running again was due to time zone difference:
```
MMEDateTests
  testHTTPDateFormatter, (([dateString isEqualToString:@"Sat, 3 Jan 1970 03:28:43 -0800"]) is true) failed - Sat, 3 Jan 1970 11:28:43 +0000
  /Users/vagrant/git/Tests/MMEDateTests.m:18
    NSString *dateString = [MMEDate.HTTPDateFormatter stringFromDate:date];
    XCTAssert([dateString isEqualToString:@"Sat, 3 Jan 1970 03:28:43 -0800"], @"%@", dateString);
}
```

For an experiment, I hard coded PST here to toy around double checking the expectation validation.

In this case, the test is expecting the string to include a time zone `Z` of PST. I'm guessing the CI box is somewhere in the UK UTC timezone as bitrise scales which may have revealed this issue. 

I'm looking to see what the expectation was for this behavior? Does this need the time zone Z? If not, we could send a timestamp formatted to a hardcoded UTC which would allow us to ensure the test passes regardless of which timezone the CI box is run.

Reply:
Good find! This should be a UTC check for sure. We’re only verifying formatting here.